ui/vendor/css/bootstrap.css

```diff
@@ -4087,7 +4527,16 @@ select[multiple].input-group-sm > .input-group-btn > .btn {
     margin-right: 15px;
     margin-left: 15px;
   }
-  .navbar-text.navbar-right:last-child {
+}
+@media (min-width: 768px) {
+  .navbar-left {
+    float: left !important;
+  }
+  .navbar-right {
+    float: right !important;
+    margin-right: -15px;
```

Comment:
Ya, we'll have to add that change above. Without it the quick filter clear [x] icon is off screen when a filter is active:

![filterclipped](https://cloud.githubusercontent.com/assets/3660661/8854868/35055226-3130-11e5-92d1-e525e01b8935.jpg)